SimFastTiming/FastTimingCommon/src/BTLBarDeviceSim.cc

```diff
@@ -84,36 +84,43 @@ void BTLBarDeviceSim::getHitsResponse(const std::vector<std::tuple<int, uint32_t
     // --- Get the simHit energy and convert it from MeV to photo-electrons
     float Npe = 1000. * hit.energyLoss() * LightYield_ * LightCollEff_ * PDE_;
 
-    // --- Get the simHit time of arrival
-    float toa = std::get<2>(hitRef);
```

Comment:
why is this removed and replaced with `std::get<2>(hitRef)` multiple times in the lines below?

Reply:
I had a second thought on introducing a variable just to store a value which was already there